index.html

```diff
@@ -10,39 +10,250 @@
+          src="./image/work2.png"
+        />
+        <h3 class="work-title">MUlti-Post Stories</h3>
+        <div class="profession">
+          <p>CANOPY</p>
+          <div class="expert">
+            <div class="bullet"></div>
+            <p>Back End Dev</p>
+          </div>
+          <div class="expert">
+            <div class="bullet"></div>
+            <p>2015</p>
+          </div>
+        </div>
+        <h3 class="description">
+          A daily selection of privately personalized reads;no accounts or
+          sign-ups required
+        </h3>
+
+          <ul class="languages">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+         
+          <button class="see-project">See project</button>
+      </section>
+      <section class="work">
+        <img
+           width="350px"
+          src="./image/work3.png"
+        />
+        <h3 class="work-title">Tonic</h3>
+        <div class="profession">
+          <p>CANOPY</p>
+          <div class="expert">
+            <div class="bullet"></div>
+            <p>Back End Dev</p>
+          </div>
+          <div class="expert">
+            <div class="bullet"></div>
+            <p>2015</p>
+          </div>
+        </div>
+        <h3 class="description">
+          A daily selection of privately personalized reads;no accounts or
+          sign-ups required
+        </h3>
+
+          <ul class="languages">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+         
+          <button class="see-project">See project</button>
+          </section>
+          <section class="work">
+            <img
+               width="350px"
+              src="./image/work4.png"
+            />
+            <h3 class="work-title">MUlti-Post Stories</h3>
+            <div class="profession">
+              <p>CANOPY</p>
+              <div class="expert">
+                <div class="bullet"></div>
+                <p>Back End Dev</p>
+              </div>
+              <div class="expert">
+                <div class="bullet"></div>
+                <p>2015</p>
+              </div>
+            </div>
+            <h3 class="description">
+              A daily selection of privately personalized reads;no accounts or
+              sign-ups required
+            </h3>
+    
+              <ul class="languages">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+              </ul>
+             
+              <button class="see-project">See project</button>
+              </section>
+              <section class="work">
+<h3 class="work-title">About my self</h3>
+<p id="about">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+<div class="social">
```

Comment:
- [ ] If you notice, the about section container has a curvered border on top right of the card which should be applied to make sure that the design matches the Figma template design.
Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/236159853-df7d70d3-6e04-4c39-8b98-a9d5dc7760bb.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/236159960-56233398-2cd6-48bc-8a9c-d98f6b4aeaa7.png)